.golangci.yml

```diff
@@ -3,30 +3,31 @@ linters-settings:
     locale: US
 
 linters:
-  # please, do not use `enable-all`: it's deprecated and will be removed soon.
-  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
-  # disable-all: true
   enable:
     - golint
     - misspell
-
-  # don't enable:
-  # - gochecknoglobals
-  # - gocognit
-  # - godox
-  # - maligned
-  # - prealloc
+    - gocyclo
+    - bodyclose
+    - unconvert
+    - goconst
+    - goimports
+    - unparam
+    - whitespace
+    - godot
```

Comment:
All exporting comments should end with a period.
Took a lot of work, but is better for godocs.